final_task/rss_reader/NewsCache.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os.path
+import logging
+
+
+class NewsCache:
+
+    def __init__(self, file_name):
+        self.logger = logging.getLogger(__name__)
+        self.file_name = file_name
+        self.logger.info("Check for cache file...")
+        if os.path.isfile(file_name):
+            try:
+                with open(file_name, "r") as json_file:
+                    self.cache_news = json.load(json_file)
+                    self.logger.info("Reading cache file...")
+            except Exception as ex:
+                self.logger.error("Error reading file: {} {}".format(type(ex), ex))
```

Comment:
It is better to use `os.path.exists` instead of this

Reply:
But above there is os.path.isfile. Why os.path.exists is better?

Reply:
You won't need to handle `FileNotFoundError` in this case

Reply:
This exception is for such error like nonreadable format of file and in this case file will be overwritten.